app/services/onlinetesting/OnlineTestService.scala

```diff
@@ -183,20 +181,19 @@ trait OnlineTestService {
   }
 
   private def emailInviteToApplicant(application: OnlineTestApplication, emailAddress: String,
-    invitationDate: DateTime, expirationDate: DateTime): Future[Unit] = {
+                                     invitationDate: DateTime, expirationDate: DateTime): Future[Unit] = {
     val preferredName = application.preferredName
     emailClient.sendOnlineTestInvitation(emailAddress, preferredName, expirationDate).map { _ =>
       audit("OnlineTestInvitationEmailSent", application.userId, Some(emailAddress))
     }
   }
 
   private def markAsInvited(application: OnlineTestApplication)
-    (onlineTestProfile: Phase1TestProfile): Future[Unit] = for {
-    _ <- otRepository.insertPhase1TestProfile(application.applicationId, onlineTestProfile)
-    _ <- appRepository.updateProgressStatus(application.applicationId, ProgressStatuses.PHASE1_TESTS_INVITED)
+                           (onlineTestProfile: Phase1TestProfile): Future[Unit] = for {
+    _ <- appRepository.insertPhase1TestProfile(application.applicationId, onlineTestProfile)
   } yield {
-      audit(s"ApplicationStatus set to ${ApplicationStatus.PHASE1_TESTS} - ProgressStatus set to" +
-        s" ${ProgressStatuses.PHASE1_TESTS_INVITED}", application.userId)
+    audit(s"ApplicationStatus set to ${ApplicationStatus.PHASE1_TESTS} - ProgressStatus set to" +
```

Comment:
the first parameter of audit is the event id, not a free text description. It should be something like OnlineTestInvited or whatever is relevant. If additional meta data is needed, it can be passed into logEventNoRequest()